### DISCUSSION TITLE
search_recent_tweets() return result sorted by chronicle order

### DISCUSSION THREAD

**Discussion by yxd0018** [tweepy/tweepy#2001] [Questions] [answered]
I use search_recent_tweets() to search all tweets by conversation_id. It takes sort_order argument but only 2 options: recency (most recent first) and relevancy .  When I search by conversation_id, even if max_results is large enough, the first tweet may still be missing due to recent less-relevant tweets. I can use from and to in query to minimize the scope and wonder if there is a direct way to get tweets in chronicle order by just conversation_id? Thanks.

**Comment by Harmon758** [comment, marked as answer]
Assuming you mean by chronological order, that's not something that [that Twitter API endpoint](https://developer.twitter.com/en/docs/twitter-api/tweets/search/api-reference/get-tweets-search-recent) supports.
You'll have to paginate through the results.

**Reply by yxd0018** [reply]
Thanks a lot. I have a follow-up question regarding the same search_recent_tweets(). With max_results=50 I get 32 tweets and still missing the first one https://twitter.com/fallacyalarm/status/1586871383673700353 
Anything I did wrong? 
As remedy, I basically manually load the first tweet whose id = conversation_id. 

In [108]: tweets = client.search_recent_tweets('conversation_id:1586871383673700353', max_results=50).data
In [109]: len(tweets)
Out[109]: 32
In [110]: tweets[-1].text
Out[110]: '2/In my investment case above ........'  <=== its 2nd tweet in the thread
